### PR TITLE
fix to subshare config and test account input

### PR DESCRIPTION
**Pull request by rrobison44**
With more than 5 test accounts the <input> line exceeded the 132 char maximum resulting in invalid html.  Trying to add a new test account would fail -- likely due to the broken html on the input.

### CODE REVIEW

File: open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.CONFIG [hunk 1]
```diff
@@ -1176,8 +1179,8 @@ PROCEDURE DISPLAYHTML
 
  HTMLVIEWLINE("<td class='lbl width300'>Test accounts list:</td>")
  HTMLVIEWLINE("<td>")
- HTMLVIEWLINE("<input id='testModeAccountList' class='width600' type='text' value='"+
-  SACTESTMODEACCOUNTLIST+"' ")
+ HTMLVIEWLINE("<input id='testModeAccountList' class='width600' type='text'")
```
Comment (tkainz) [inline open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.CONFIG:1182]: This fix is good as it currently is but just to be safe, should we add a maxlength='109' to the input to limit the user input to 10 account numbers and 9 delimiters between them?

**Review by tkainz — APPROVED** [review]
added additional suggestion to fix, but not required.

File: open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.CONFIG (hunk 1 above)
Comment (tkainz) [reply open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.CONFIG:1182]: Also.... have you checked the other input fields to make sure there are no similar potential issues?

**Review by jordan-keenan — APPROVED** [review]
Code Fix looks good!
I think Tom's Idea is also good, if you have time to implement that.

File: open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.CONFIG (hunk 1 above)
Comment (rrobison44) [reply open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.CONFIG]: added maxlength
